### ISSUE TITLE
Message::sendNotificationEmail - notification_address_from / notification_address_from_name

### ISSUE THREAD

**Issue by pavsid** [jan-vince/smallcontactform#94]
The 'from name' for the notification email is set to be the same as the 'from email'. From what I see, there is currently no field in the backend to set the 'from name', however we are using this method externally and looking at the code it looks like this should be supported anyway.

Suggest to change the following:

```
/**
 * Set From address if defined in component property
 */
if ( !empty($componentProperties['notification_address_from'])) {

    if (!empty($componentProperties['notification_address_from'])) {
        $fromAddressName = $componentProperties['notification_address_from'];
    } else {
        $fromAddressName = null;
    }

    $message->from($componentProperties['notification_address_from'], $fromAddressName);
}

});
```

...to...

```
/**
 * Set From address if defined in component property
 */
if ( !empty($componentProperties['notification_address_from'])) {

    if (!empty($componentProperties['notification_address_from_name'])) {
        $fromAddressName = $componentProperties['notification_address_from_name'];
    } else {
        $fromAddressName = null;
    }

    $message->from($componentProperties['notification_address_from'], $fromAddressName);
}
```

**Comment by jan-vince** [comment]
Hi @pavsid good idea - I will add this to the next release.

Thanks!

**Comment by jan-vince** [comment]
Hi @pavsid, this is fixed in 1.40.1. 

Thanks a lot!
